<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DNSpecBath</class>
 <widget class="QMainWindow" name="DNSpecBath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оценка глубины прибрежных территорий</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>491</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>769</width>
       <height>569</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="DNWidgetImage" name="DNWPic" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="WidgetViewParam" native="true">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>171</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="LayRGB" rowstretch="0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>7</number>
     </property>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="ButtonReCalcImg">
       <property name="text">
        <string>Создать изображение</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QTreeWidget" name="treePolygons">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,50,0" rowminimumheight="0,0,0,0" columnminimumwidth="0,50,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="verticalSpacing">
        <number>7</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>R:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>G:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="EditKG">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="EditKB">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="EditKR">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="ComboB">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ComboG">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ComboR">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Параметры изображения</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Полигоны интереса</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="pbVectorize">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Векторизовать результат</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="FileOpen"/>
    <addaction name="OpenLastImg"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Полигон</string>
    </property>
    <addaction name="CreatePoly"/>
    <addaction name="separator"/>
    <addaction name="MultiChan"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Тематические задачи</string>
    </property>
    <addaction name="Batinometriy"/>
    <addaction name="ShowDeptch"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="FileOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="CreatePoly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Создать полигон</string>
   </property>
  </action>
  <action name="Batinometriy">
   <property name="text">
    <string>Батинометрия</string>
   </property>
  </action>
  <action name="OpenLastImg">
   <property name="text">
    <string>Открыть последнее изображение</string>
   </property>
  </action>
  <action name="MultiChan">
   <property name="text">
    <string>Множитель каналов</string>
   </property>
  </action>
  <action name="ShowDeptch">
   <property name="text">
    <string>Показать значения глубин</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DNWidgetImage</class>
   <extends>QWidget</extends>
   <header location="global">dnwidgetimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
